cmake/FindLIBHWLOC.cmake

```diff
@@ -25,9 +42,21 @@ if(LIBHWLOC_LIBRARY)
     message(STATUS "    LIBHWLOC_LIBRARIES = ${LIBHWLOC_LIBRARIES}")
     message(STATUS "    LIBHWLOC_INCLUDE_DIRS = ${LIBHWLOC_INCLUDE_DIRS}")
     message(STATUS "    LIBHWLOC_LIBRARY_DIRS = ${LIBHWLOC_LIBRARY_DIRS}")
+    message(STATUS "    LIBHWLOC_VERSION = ${LIBHWLOC_VERSION}")
     if(WINDOWS)
         message(STATUS "    LIBHWLOC_DLL_DIRS = ${LIBHWLOC_DLL_DIRS}")
     endif()
+
+    if(LIBHWLOC_FIND_VERSION)
+        if(NOT LIBHWLOC_VERSION)
+            message(WARNING "Failed to retrieve libhwloc version")
```

Comment:
it looks like we got a lot of warnings in Windows jobs

Reply:
Should be resolved now.